### PR TITLE
Splash screen

### PR DESCRIPTION
**Pull request by ajilantang**
- [X]  check token and switch to events  routes or login screen 
- [X]  refactor login's feature #50

### CODE REVIEW

File: src/features/auth/screens/LoginScreen.js [hunk 1]
```diff
@@ -275,6 +263,7 @@ type StateToProps = {
 };
 type Dispatch = (action: Object) => void;
 export function mapStateToProps(state: StateToProps) {
+  console.log('mapstatelogin');
```
Comment (jhnoa) [inline src/features/auth/screens/LoginScreen.js:266]: pls remove

**Review by jhnoa — APPROVED** [review]

**Comment by ajilantang** [comment]
Initial  routes (splash screen) and would check user logged in or not 

screen shot 2018-08-31 at 11 58 50

if user has been logged in and token valid would redirect to this page . [home screen]

screen shot 2018-08-31 at 11 59 22

else

screen shot 2018-08-31 at 12 00 00

its also save current user information (async storage ['currentUser'])

**Comment by ajilantang** [comment]
- [X] Add signOut  
- [X] fixed bugs splash screen 😄 , async storage error because set null at async storage .  
ref #50 and #34  api for login 
- [ ] remove cookies needed help . cookies manager doesnt work , i dont know how to used cookies manager in expo

**Comment by ajilantang** [comment]
ezgif com-crop
 remove cookies needed help . cookies manager doesnt work , i dont know how to used cookies manager in expo but its has been removed token from asyncstorage .

**Pull request merged**